src/dev.ts

```diff
@@ -23,10 +23,15 @@ export const handle = (userApp?: Hono, options?: Options) => {
     const args = createGetLoadContextArgs(c)
 
     const reactRouterContext = getLoadContext(args)
-    return handler(
-      c.req.raw,
-      reactRouterContext instanceof Promise ? await reactRouterContext : reactRouterContext
-    )
+    const resolvedContext = reactRouterContext instanceof Promise ? await reactRouterContext : reactRouterContext
+
+    const response = await handler(c.req.raw, resolvedContext)
+
+    if (response.headers.get('content-type')?.includes('text/html')) {
+      response.headers.set('transfer-encoding', 'chunked')
```

Comment:
I'm not sure it's a good design if we set `transfering-encoding: chunked` for all HTML responses.

Reply:
I agree, this fix was quite hasty. Since it doesn't seem to be a straightforward fix, I will log an issue for it at this time.